### ISSUE TITLE
Mobile title screen bug

### ISSUE THREAD

**Issue by imjustredrlol** [JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag#48]
### Describe your bug here.

Whenever i go on js engine mobile, it just crashes. I put the stuff in the folder etc. It wont work...

### Command Prompt/Terminal logs (if existing)

```bash
Uncaught error! Null Object Reference.
```

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Neko, HashLink, or other build system

### Did you edit anything in this build? If so, mention or summarize your changes.

No.

**Comment by JordanSantiagoYT** [comment]
yeah, I didn't make the Android port so nothing i can do to help you here

**Comment by imjustredrlol** [comment]
> yeah, I didn't make the Android port so nothing i can do to help you here

Oh well, i wonder who did..

**Comment by Stefan2008Git** [comment]
> > yeah, I didn't make the Android port so nothing i can do to help you here
> 
> Oh well, i wonder who did..

That's me bruh. I am nothing litterally. I am not the best Android porter, okay? Since Joolian is so freaking lazy to ask @MaysLastPlay10 to try to ask to fix Android issue instead of me because

**Comment by JordanSantiagoYT** [comment]
fixed in 1.13.0
